packages/ckeditor5-bookmark/src/bookmarkediting.ts

```diff
@@ -88,6 +94,15 @@ export default class BookmarkEditing extends Plugin {
 				return toWidget( containerElement, writer, { label: labelCreator } );
 			}
 		} );
+
+		conversion.for( 'upcast' ).elementToElement( {
+			view: element => upcastMatcher( element ),
+			model: ( viewElement, { writer } ) => {
+				const bookmarkId = viewElement.getAttribute( 'id' );
```

Comment:
what about `name`, or we will manage that in next iteration?

Reply:
Not in this iteration.